### PR TITLE
Update to ALCARECO MuonAlignment producers

### PR DESCRIPTION
**Pull request by anthonykrose**
This includes several changes to the ALCARECO Producers used by during MuonAlignment (changed thresholds, added several new collections to the output files, changed some of the detector status flags and fixed a bug that existed in the Overlaps stream during Run-1).

### CODE REVIEW

**Comment by diguida** [comment]
please test

File: Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_cff.py [hunk 1]
```diff
@@ -15,7 +29,9 @@
 # DT calibration needs as many muons with DIGIs as possible, which in cosmic ray runs means standAloneMuons
 #    nHitMinGB = 1, # muon collections now merge globalMuons, standAlone, and trackerMuons: this stream has always assumed globalMuons
     ptMin=cms.double(0.),
-    pMin=cms.double(10.)
+    pMin=cms.double(25.)
```
Comment (mmusich) [inline Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_cff.py]: @anthonykrose a  "," is missing here, therefore causing compilation errors. Please add it.

Comment (anthonykrose) [inline Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_cff.py]: Sorry - should have caught that when compiling. Just commited the fix.

**Comment by mmusich** [comment]
@cmsbuild please acknowledge changes

**Comment by mmusich** [comment]
please test

File: Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_Output_cff.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
     ),
     outputCommands = cms.untracked.vstring(
         'keep *_ALCARECOMuAlCalIsolatedMu_*_*', 
+        'keep *_generalTracks_*_*',
```
Comment (mmusich) [inline Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_Output_cff.py]: @anthonykrose @diguida are generalTracks really needed? I thought `ALCARECOMuAlCalIsolatedMu` contained global muons already

Comment (anthonykrose) [inline Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_Output_cff.py]: This was a request from Yuriy based on some changes he plans to make to the way the track propagation is handled in the Muon Alignment procedure. I haven't seen his code (and he isn't in the office) so I would hesitate commenting further before speaking to him @pakhotin

Comment (mmusich) [inline Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_Output_cff.py]: @pakhotin @anthonykrose any update on the question above and the estimate of the events size?
Thanks

Comment (anthonykrose) [inline Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_Output_cff.py]: I am working on the RelVal stuff - I required some input from PPD conveners so it was delayed a few days. Assuming my current tests work I should have an answer by today/tomorrow conerning the event size. Yuriy is back in the office today too so I'll chat to him about the generalTracks questions.

**Comment by diguida** [comment]
@cmsbuild please test

File: Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_Output_cff.py (hunk 2 above)
Comment (diguida) [inline Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_Output_cff.py]: @anthonykrose @pakhotin
As discussed in the workshop, the `generalTracks` is *needed* for track propagation in Muon chambers. As reported by @abbiendi, you should add a selector on track transverse momentum by adding it to the AlCaReco `Path`.

**Comment by diguida** [comment]
@anthonykrose any news on the event size? Have you tried the selector on track pt?

**Comment by mmusich** [comment]
@anthonykrose @pakhotin we understand this has probably low priority, but to move on we really need to know if you foresee updates on the track pt selection as suggested by @abbiendi. 
This PR is now 50 days old.

**Comment by anthonykrose** [comment]
Hi guys, sorry for the slow response on this - I've been working on some other pressing issues related to the alignment but now these appear to be resolved. I'll aim to have this done and committed in the coming days.

**Comment by mmusich** [comment]
-1
superseded by #9482

**Comment by mmusich** [comment]
@cmsbuild please close this one

**Pull request closed without merging**